libs/sdk-ui-kit/styles/scss/variables.scss

```diff
@@ -69,6 +69,9 @@ $gd-palette-success-dimmed: var(
 // secondary colors
 $gd-color-positive-dimmed: mix($gd-color-positive, $default-gd-color-text-light, 10%);
 
+// moved from goodstrap, this variable is used on KD
+$gd-color-negative-dimmed: mix($gd-color-negative, $default-gd-color-text-light, 10%);
```

Comment:
Should we rather move it just to KD?

Reply:
And is it used in SDK at all? If not so, maybe we should specify it just in KD? (there are also other KD-only variables, or not?)

Reply:
They are not used on SDK, will move them to KD.